CONTRIBUTING.md

```diff
@@ -120,6 +120,17 @@ You can run samples in a real device without changing certificates and provision
 
 This will setup certificates and provisioning profiles into your machine, but in order to be able to run a sample in a real device you need to register that device with Sentry AppConnect account, add the device to the provisioning profile you want to use, download the profile again and open it with Xcode.
 
+## Promoting a beta release to a normal release
+
+We frequently release a beta version of our SDK and dogfood it with internal apps to increase our SDK stability. We continue to merge PRs to the main branch, so we can't promote a beta release by publishing it from the main branch. Instead, we create a branch from the GH tag of the beta release and promote it from there. To do this, follow these steps:
+
+1. Checkout a new branch from the GH tag of the beta release: `git checkout -b publish/x.x.x x.x.x-beta.1`. You can't use `release/x.x.x` or `x.x.x` as the branch name as craft will fail, as it creates a `release/x.x.x` branch for updating the changelog and it will create a tag `x.x.x` for the release.
+2. Duplicate the Changelog.md entry of the beta release and change header of the version number to unreleased.
+3. Commit and push the changes.
+4. Trigger the release workflow with use workflow from the `publish/x.x.x` branch.
```

Comment:
m: During the release workflow, we can choose the same branch for Craft to merge it back, so Craft does not fail.
Also,  we need to manually update the changelog in the main branch.

Reply:
But that sucks pretty much. Ideally, we would merge the branch back to main. Does it then fail if we do that?

Reply:
It fails because more likely than not, the main will be "ahead" of the beta tag.

Reply:
OK, but then I would say let crash merge back into the publish branch and then merge the publish branch manually back to main?

Reply:
Yes, we can do that after merging main into the publish branch. Good idea

Reply:
I'm also for setting the target as `publish/x.x.x` for the `x.x.x` release of `x.x.x-beta.x`. 

And then merging manually, if merged directly to mainl, it will fail in better case, in worse case it will break changelog or code.

Not ideal but safe. 

Note: The JS team solved this by using `develop` branch as default and main for releases. Maybe we could adopt similar workflow in the future.

Reply:
I updated the description. @brustolin and @krystofwoldrich please have another look.